### PR TITLE
Upgrade Wasmer dependency from 2.0.0 to 2.3

### PR DESCRIPTION
**Pull request by miloszm**
Upgrade Wasmer dependency from 2.0.0 to 2.3

Solves issue #351

### CODE REVIEW

File: rusk-uplink/Cargo.toml [hunk 1]
```diff
@@ -7,6 +7,6 @@ edition = "2018"
 
 [dependencies]
 microkelvin = { version = "0.16.0-rkyv", default-features = false }
-rkyv = { version = "0.7.26", default-features = false, features = ["alloc", "validation"] }
+rkyv = { version = "0.7.29", default-features = false, features = ["alloc", "validation"] }
```
Comment (ureeves) [inline rusk-uplink/Cargo.toml:10]: Is there a reason we want to specify the minor version? I care because this is likely to become a public crate.

**Review by ureeves — CHANGES_REQUESTED** [review]
One version dependency version issue, but looks good!

**Review by krl — APPROVED** [review]
lgtm

File: rusk-uplink/Cargo.toml (hunk 1 above)
Comment (miloszm) [reply rusk-uplink/Cargo.toml:10]: here I'd be rather careful, I think we don't want to progress with rkyv versions without losing control at the moment, at least not at this stage yet, I'd keep this thought for the future though

Comment (ureeves) [reply rusk-uplink/Cargo.toml:10]: Alright. Lets add a comment to revisit the subject at a later date.

**Pull request merged**